### ISSUE TITLE
Failed to parse ("Dividends", "Data", "EUR", ...

### ISSUE THREAD

**Issue by Rom888** [KonishchevDmitry/investments#36]
```
$ investments tax-statement ib
W: The broker statement misses trade settle date information. First occurred trade - EXH4d at 16.04.2020. All calculations for such trades will be performed in T+0 mode.
E: Error while reading "/media/t/fin/investments-data/en-activity-U3594789_2020_2020.csv" broker statement: Failed to parse ("Dividends", "Data", "EUR", "2020-07-15", "EXH4(DE000A0H08J9) Cash Dividend EUR 0.013046 per Share (Mixed Income)", "1.71") record: Unexpected dividend description: "EXH4(DE000A0H08J9) Cash Dividend EUR 0.013046 per Share (Mixed Income)".
```

I have the following entries in the description columns:
```
EXH4(DE000A0H08J9) Cash Dividend EUR 0.013046 per Share (Mixed Income)
EXH4(DE000A0H08J9) Cash Dividend EUR 0.162923 per Share (Mixed Income)
EXV8(DE000A0H08F7) Cash Dividend EUR 0.140474 per Share (Mixed Income)
SOXX(US4642875235) Cash Dividend USD 1.069314 per Share (Ordinary Dividend)
SOXX(US4642875235) Cash Dividend USD 0.746891 per Share (Ordinary Dividend)
```

**Comment by KonishchevDmitry** [comment]
Thanks, fixed in 2.11.2.

**Comment by Rom888** [comment]
Updated, it works now, thank you, great software!
